NEXTCLOUD.cmake

```diff
@@ -15,6 +15,28 @@ set( APPLICATION_VIRTUALFILE_SUFFIX "nextcloud" CACHE STRING "Virtual file suffi
 set( LINUX_PACKAGE_SHORTNAME "nextcloud" )
 set( LINUX_APPLICATION_ID "${APPLICATION_REV_DOMAIN}.${LINUX_PACKAGE_SHORTNAME}")
 
+option(DEV_BUILD "If enabled, name variables will be appended with safe variations of DEV_SUFFIX in order to reduce conflicts." OFF)
```

Comment:
Maybe it's better to describe what it does and not how. Like: If enabled the build will use a different configuration directory and application id to ...